control-plane/roles/valkey/templates/valkey-secret.yaml

```diff
@@ -0,0 +1,12 @@
+{% if valkey_registry_auth_enabled %}
```

Comment:
The Ansible task should have this condition in a `when` otherwise an empty yaml file would be applied.